leaderboardPage.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./normalize.css"/>
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <script src="./script.js" defer></script>
+    <script src="./leaderboardScript.js" defer></script>
+    <title>Surfwave Fury Leaderboard</title>
+</head>
+<body>
+<main>
+    <div class="leaderboardTopContainer">
+        <div class="leaderboardTitleContainer">
+            <h1 class="leaderboardTitle">Surfwave Fury</h1>
+        </div>
+        <div class="backButtonContainer">
+            <button id="backButton">Back to Surfwave Fury</button>
+        </div>
+    </div>
+    <div class="leaderboardContainer">
+        <div id="mainLeaderboardContainer">
+            <div class="leaderboardSurfboardContainer">
+                <img class="leaderboardSurfboardImage" src="./imgs/surfboardLeaderboard.png" alt="surfboard"/>
+                <div class="leaderboardSurfboardTextContainer">
+                    <h2 class="leaderboardSubTitleOne">Leader</h2>
+                    <h2 class="leaderboardSubTitleTwo">Board</h2>
```

Comment:
instead of having 2 different `h2`s, better to have it as one with two `span`s inside of it. then if you apply those classes to the `span`s it will still work. better because they are both part of the same subheading, so better for accessibility, search engines etc